contracts/Conference.sol

```diff
@@ -172,9 +177,46 @@ contract Conference is Destructible, GroupAdmin {
      * @return True if the user is marked as attended by admin.
      */
     function isAttended(address _addr) public view returns (bool){
-        return isRegistered(_addr) && participants[_addr].attended;
+        if (!isRegistered(_addr)) {
+            return false;
+        }
+
+        // check the attendance maps
+        if (0 < attendanceMaps.length) {
+            Participant storage p = participants[_addr];
+            uint pIndex = p.participantIndex - 1;
+            uint map = attendanceMaps[uint(pIndex / 256)];
+            return (0 < (map & (2 ** (pIndex % 256))));
+        }
+
+        return participants[_addr].attended;
```

Comment:
Maybe a bit pedantic but you may want to use `if(){}else{}` style with comment that this is old way to be in consistent with your `totalAttended()` function (or take `else` from `totalAttended()` )

```
if (0 < attendanceMaps.length) {
    Participant storage p = participants[_addr];
    uint pIndex = p.participantIndex - 1;
    uint map = attendanceMaps[uint(pIndex / 256)];
    return (0 < (map & (2 ** (pIndex % 256))));
}else{
    return participants[_addr].attended;
}
```